package.json

```diff
@@ -121,91 +121,92 @@
     "manage:translations":
       "yarn workspace blockchain-wallet-v4-frontend manage:translations"
   },
-  "dependencies": {},
+  "dependencies": {
+    "compression": "https://registry.npmjs.org/compression/-/compression-1.7.2.tgz",
+    "onetime": "https://registry.npmjs.org/onetime/-/onetime-1.1.0.tgz"
+  },
```

Comment:
These were the two indirect dependencies that were being installed over HTTP. This is sort of a brittle hack but ensures that they get installed over HTTPS via the `yarn.lock` file.